CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 23.3.2 [#708](https://github.com/openfisca/openfisca-core/pull/708)
+
+- Addresses the typecasting from `float32` to `float64` made when outputing numbers in the API and in `openfisca-run-test`.
```

Comment:
The first and most obvious thing reading the Changelog should be the impact and users (your second point), before the technicality of how we fixed the issues (your first point).

I'd also add a small example of what changed, (maybe a short extract of a JSON?) to help illustrate what has changed 🙂